assets/javascripts/kitten/components/layout/horizontal-stroke/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react'
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+import styled, { css } from 'styled-components'
+import COLORS from '../../../constants/colors-config'
+import { pxToRem } from '../../../helpers/utils/typography'
+
+export const StyledHorizontalStroke = styled.span`
+  display: block;
+  border: none;
+  background: ${COLORS.font1};
+
+  ${({ modifierSize }) => {
+    switch (modifierSize) {
+      case 'tiny':
+        return TINY
+      case 'big':
+        return BIG
+      case 'huge':
+        return HUGE
+      default:
+        return DEFAULT
```

Comment:
Une toute petite suggestion :)

```suggestion
      case 'tiny':
      case 'big':
      case 'huge':
        return modifierSize.toUpperCase()
      default:
        return DEFAULT
```

Reply:
En fait comme on a le `PropTypes.oneOf(['tiny', 'default', 'big', 'huge']),` on n'a même pas besoin du `default: return DEFAULT`, juste du `return modifierSize.toUpperCase()`, non ?

Reply:
Le problème est que `modifierSize.toUpperCase()` retourne un `string` et non le contenu de la variable.